apps/dotcom/client/src/tla/components/TlaSidebar/components/TlaSidebarUserLink.tsx

```diff
@@ -15,15 +16,21 @@ export function TlaSidebarUserLink() {
 	if (!user) return null
 
 	return (
-		<TlaAccountMenu source="sidebar">
-			<button
-				className={classNames(styles.user, 'tla-text_ui__regular')}
-				title={accountMenuLbl}
-				data-testid="tla-sidebar-user-link"
-			>
-				<TlaAvatar img={user.avatar} />
-				<div className={classNames(styles.userName, 'notranslate')}>{user.name}</div>
-			</button>
-		</TlaAccountMenu>
+		<div
+			className={classNames(styles.user, 'tla-text_ui__regular')}
+			title={accountMenuLbl}
+			data-testid="tla-sidebar-user-link"
+		>
+			<TlaAvatar img={user.avatar} />
+			<div className={classNames(styles.userName, 'notranslate')}>{user.name}</div>
+			<TlaAppMenu>
+				<div
+					data-testid="tla-sidebar-app-menu"
+					className={classNames(styles.accountMenuTrigger, styles.hoverable)}
+				>
+					<TlaIcon icon="question" />
```

Comment:
the question mark icon feels totally wrong here, really strange pattern for a menu that includes sign out and preference settings, can we use three dots or something else? Also not sure about the need for a border on the hit target, feels a bit heavy.

Reply:
looking at other apps that have this same pattern, like figma and linear, it's not directly next to the user name in those apps so it feels less jarring to me. But I dunno, maybe I was just used to this being a user menu and other people won't notice 🤷🏼

Reply:
yes, please no question mark, it's super weird. heavy-plus-1 as the kids say

Reply:
I think we'll want to split the account-related items separate from the help menu. Let me do that. Help icon is not weird, plenty of apps are using that pattern.

<img width="452" alt="image" src="https://github.com/user-attachments/assets/8b533598-1e51-4d0b-8790-dbaa7ecc49aa" />

<img width="380" alt="image" src="https://github.com/user-attachments/assets/fff7a609-5860-4be7-99f4-6eccb071f448" />

<img width="417" alt="image" src="https://github.com/user-attachments/assets/92859f5d-61ad-474c-8fa1-435819aa961f" />

<img width="391" alt="image" src="https://github.com/user-attachments/assets/ff2fbd07-fd42-4c5a-932b-faf4deb58188" />

Reply:
>   plenty of apps are using that pattern.

hah, that's comparing apples to oranges here, the apps you're presenting are strictly help menus, not help+account hybrid menus.
notion's for example:
![Screenshot 2025-05-08 at 16 57 34](https://github.com/user-attachments/assets/f24f69c3-856d-4494-91aa-5a7217cb78f3)

whereas ours has Sign out, Theme, Language, Import - i would _never_ expect to find those things in a [?] menu!

Reply:
Ok, splitting these into two menus. I'm still hoping to add a proper account modal instead of this menu, however it does the job for now. IMO this is a little less discoverable than a garbage bag of options and things.

<img width="286" alt="image" src="https://github.com/user-attachments/assets/9f30f6a1-3ad0-4574-9c3d-279fdb3498ae" />

<img width="300" alt="image" src="https://github.com/user-attachments/assets/1e752dd6-c8b4-4610-a826-d93e5329e3a1" />